api/users.go

```diff
@@ -29,6 +29,7 @@ const (
 	shadowUser                     = userIDPath + "/federated-identity/:provider"
 	expiredToUAcceptancePath       = adminRootPath + "/expired-tou-acceptance"
 	getSupportInfoPath             = adminRootPath + "/support-infos"
+	getTrustIDAuthToken            = "/auth/realms/:realm/trustid-auth-token/realms/:realm/users/:userId/generate"
```

Comment:
getTrustIDAuthToken ? You named it "get" but the path tells it generates something... we might find a better name...
You could have to distinct realmNames in your path. It let you provide the same value twice or use 2 distincts realms.